packages/valora-auth/firebase-env-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+firebase functions:config:set \
+transak.url_staging=$(grep TRANSAK_URL_STAGING .env | cut -d '=' -f 2-) \
```

Comment:
Not really necessary but we could have a bash function that receives the env variable name as parameter and runs `grep $1 .env | cut -d '=' -f 2-` and call it in all these lines to make them slightly more readable.